### ISSUE TITLE
"This add-in could not be started." message on failing to start our Excel add-in starting this Monday

### ISSUE THREAD

**Issue by cidev5** [OfficeDev/office-js#2536]
Starting this Monday our internal users have been experiencing being unable to use our add-in within Excel desktop application (works in office.com).

It is not just our add-in that we are having issues with but also the Microsoft Dynamics add-in.

If we run Excel in Windows 8 compatibility mode by turning this option on in the Excel shortcut properties -> Compatibility tab, we can use these add-ins again.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: Version 2203 Build 16.0.15028.20178
* Operating System: Windows 10 Enterprise 21H2
* Browser (if using Office on the web): Chrome
 
## Expected behavior

Both add-ins should open and run as they always have.

## Current behavior

Add in fails to start/insert.

![add-in-error](https://user-images.githubusercontent.com/22505624/163418767-229d9f03-a1eb-4e98-b819-d7be1c3525de.png)

## Steps to reproduce

1. Recreate our OS/Excel version environment
2. Attempt to download/install our add-in from the App store

**Comment by keyur32** [comment]
Ack. Thank you for the report. Investigating.

**Comment by keyur32** [comment]
Internal Issue #301966282

**Comment by zlatko-michailov** [comment]
Hi @cidev5. Thank you for reporting this issue.

In order to repro it, we need a little more information. If this add-in is in the store, please provide its store asset ID or its full name. If it is not in the store, can you describe what Office features it leverages. In this latter case, if you can provide a sample manifest, __STRIPPED OFF OF ANY SENSITIVE INFORMATION__, that reproes it, that would very helpful.

**Comment by cidev5** [comment]
The asset ID is WA104381811

**Comment by cidev5** [comment]
Are there any thoughts about what this could be and are there any things we can do ourselves to prevent this from occurring for potentially a lot of our end users?

**Comment by zlatko-michailov** [comment]
I tried to repro the issue on 2 different machines, and I couldn't. We'll need you to collect a log and to send it to us. I'll come back with instructions.

**Comment by keyur32** [comment]
Hey @cidev5,  

A couple of suggestions:

a) Do you have [support](https://support.microsoft.com/en-us/contactus) that you can connect through your Organization?  Filing a ticket will allow a Microsoft support engineer to work with you to securely collect these logs from your machine. There are instructions available in our [April 2021 Community call](https://www.youtube.com/watch?v=hGVdftCzs20).

b) If not, please Send a frown from Excel directly, and in the description right out "OFFICEDEV GITHUB ISSUE 2536" and include your email address. 

This will allow us to contact you and we can work to get the logs directly from you in that case.

**Comment by cidev5** [comment]
I went ahead and sent a frown from Excel with the description as you described.

**Comment by keyur32** [comment]
Thanks, I don't see it yet in our system unfortunately yet. Will keep checking.

**Comment by keyur32** [comment]
Still not seeing it. Can you please try again. Also to expediate, are you able to file a support ticket? That will likely be the best way for us to securely access the logs.

**Comment by cidev5** [comment]
I tried again on Friday, and today I submitted an assistance request through the FastTrack dashboard. Let me know if there is another way to submit a support ticket as I wasn't able to find any other way that allowed me.

**Comment by keyur32** [comment]
Thanks for submitting a support ticket; I think that will be the best way to resolve this (will ping our support folks on our end as well).

**Comment by cidev5** [comment]
Hi, still trying to see if we can get to the bottom of what is causing/has caused this.

But today I reinstalled Microsoft Edge WebView2 Runtime (version 100.0.1185.44) after having uninstalled it several days ago, and the add-in now works again. I suspect a version prior had the issue (100.0.1185.36), and it has since been fixed with this update.

It is strange however that the issue was occurring even when I had the runtime uninstalled.
